platform/app/src/components/ViewportGrid.tsx

```diff
@@ -17,6 +17,17 @@ function ViewerViewportGrid(props) {
     servicesManager as ServicesManager
   ).services;
 
+  /**
+   * Determine whether users need to use the tools directly, or whether they need to click once to activate the viewport before using tools.
+   * If 'activateViewportBeforeInteraction' is available in the 'window.config' object, use its value;
+   * otherwise, default to true.
+   * If true, users need to click once to activate the viewport before using the tools.
+   * if false, tools can be used directly.
+   */
+  const activateViewportBeforeInteraction = useMemo(() => {
+    return window.config?.activateViewportBeforeInteraction ?? true;
```

Comment:
use the appconfig instead of window. and also please add this option to the configuration documentation

Thanks